cmd/bosun/expr/funcs.go

```diff
@@ -1182,6 +1199,51 @@ func Rename(e *State, T miniprofiler.Timer, set *Results, s string) (*Results, e
 	return set, nil
 }
 
+// csv is a shorthand for strings.Split(s, ",")
+func csv(s string) []string {
+	return strings.Split(s, ",")
+}
+
+func MetaTag(e *State, T miniprofiler.Timer, set *Results, lookupTagsCSV, keyCSV string) (*Results, error) {
+	keys := csv(keyCSV)
+	res := Results{}
+	lookupTags := make(map[string]struct{})
+	for _, k := range csv(lookupTagsCSV) {
+		lookupTags[k] = struct{}{}
+	}
+	for _, result := range set.Results {
+		ts := make(opentsdb.TagSet)
+		for k := range lookupTags {
+			if value, ok := result.Group[k]; ok {
+				ts[k] = value
+				continue
+			}
+			return nil, fmt.Errorf("tag key %v not found in result", k)
+		}
+		mds, err := e.Metadata.GetMetadata("", ts)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, md := range mds {
+			if md.Name == keys[0] {
```

Comment:
Do we need to detect multiple different values returned (just looking up iface data by host)? zero values returned?